generators/server/templates/_pom.xml

```diff
@@ -1200,7 +1209,7 @@
                 <!-- log configuration -->
                 <logback.loglevel>INFO</logback.loglevel>
                 <!-- default Spring profiles -->
-                <spring.profiles.active>prod</spring.profiles.active>
+                <spring.profiles.active>prod${profile.no-liquibase}</spring.profiles.active>
```

Comment:
Ah! Good call! I read in the docs that no-swagger has no effect in prod, so I excluded it here. But now I see that gradle also includes it for prod.

I will add it when I get to campus. When I adapt the documentation, should I mention this option there as well?